testing/src/main/java/org/motechproject/nms/testing/service/impl/TestingServiceImpl.java

```diff
@@ -244,17 +257,37 @@ public Object execute(Query query) {
                 return null;
             }
         };
-        stateDataService.executeSQLQuery(sqe);
+        try {
+            stateDataService.executeSQLQuery(sqe);
+        } catch (Exception e) {
+            String s = String.format("Exception While executing \"%s\"", sqlQuery);
+            LOGGER.error(s);
+            throw e;
+        }
     }
 
 
-    private void disableConstraints() {
-        changeConstraints(true);
-    }
+    private void truncateTable(final String table) {
+        SqlQueryExecution sqe = new SqlQueryExecution() {
 
+            @Override
+            public String getSqlQuery() {
+                return String.format("DELETE FROM %s WHERE 1=1", table);
```

Comment:
Sorry I'm late to the party.  Do you want to use delete or truncate?  I think truncate would be faster.

https://dev.mysql.com/doc/refman/5.0/en/truncate-table.html